classes/PodsInit.php

```diff
@@ -554,12 +556,23 @@ public function register_assets() {
 				'jquery-ui-core',
 				'jquery-ui-sortable',
 				'pods-i18n',
-				'marionette',
+				'pods-marionette',
 				'media-views',
 				'media-models',
 			), PODS_VERSION, true
 		);
 
+		/**
+		 * Page builders.
+		 */
+		if (
+			doing_action( 'elementor/editor/before_enqueue_scripts' ) || // Elementor
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 705b88bf92c18c692f2ff6d885551f55</sub>